woocommerce/woocommerce-gutenberg-products-block
# Fix product preview bugs

#57 should be merged before this is reviewed, as this PR builds off the changes from #57.

- Fixes #35.
- Block preview now looks correct from 1-6 columns, including the Add to Cart button.
- Column min/max/default settings are now pulled from the theme when possible.
- Fixes #58
- Closes #61

To test:
- Make a new product block
- Observe the grid/list toggle is not visible until you've selected a scope.
- In preview mode, verify everything looks good for 1-6 rows/columns.
